Automattic/studio
# Studio: Ensure that import is not started too early

Related to https://github.com/Automattic/dotcom-forge/issues/8547

## Proposed Changes

This PR ensure that the import queries run after the import is finished so that WP-admin auto-login and the enabled theme work as expected and the site does not need to be stopped and started. It also fixes an issue with the import state not being set correctly for a new site that is importing.

## Testing Instructions

* Pull the changes from this branch
* Download custom [wp-cli.phar here](https://github.com/user-attachments/files/16456950/wp-cli.phar.zip).
* Unzip and place the wp-cli.phar file inside of the app data's `server-files` directory. Overwrite the existing one if necessary.
* Run `nvm use && npm install && STUDIO_IMPORT_EXPORT=true npm run start` to start Studio.
* Click on `Add site` in the sidebar and add an import file (Suggestion: Try to set a custom theme on the export site first so that you can easily confirm that the import is successful)
* Start adding the site
* Observe that after the import is completed, the site is started and you can log into WP-Admin without being asked to log in
* Observe that the site thumbnail is immediately updated when the import is finished to a new theme


## Pre-merge Checklist
